### ISSUE TITLE
NPM package install failure on v5.44.0

### ISSUE THREAD

**Issue by AaronFriel** [pulumi/pulumi-azure#1301]
### What happened?

On installing `@pulumi/azure` with NPM, an error is thrown:

```
npm ERR! code 1
npm ERR! path /tmp/tmp.h3sZ4M7wsD/node_modules/@pulumi/github
npm ERR! command failed
npm ERR! command sh -c node scripts/install-pulumi-plugin.js resource azure v5.44.0
npm ERR! node:internal/modules/cjs/loader:1078
npm ERR!   throw err;
npm ERR!   ^
npm ERR!
```

### Expected Behavior

Expect `npm install` to succeed.

### Steps to reproduce

In a Node project, `npm install @pulumi/azure@v5.44.0`

### Output of `pulumi about`

n/a

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by AaronFriel** [comment]
Resolved in the latest release.

**Comment by t0yv0** [comment]
Do we need a postmortem here? If so could you link? Thank you. CC @mikhailshilkov

**Comment by AaronFriel** [comment]
Yeah, we should do a postmortem against the pulumi-publisher issue (root cause, P0 worthy due to scope).
